src/Generator/Security/BasicAuthenticationSecurityStrategy.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Generator\Security;
+
+use DoclerLabs\ApiClientGenerator\Ast\Builder\CodeBuilder;
+use DoclerLabs\ApiClientGenerator\Entity\ImportCollection;
+use DoclerLabs\ApiClientGenerator\Entity\Operation;
+use DoclerLabs\ApiClientGenerator\Input\Specification;
+use DoclerLabs\ApiClientGenerator\Naming\CopiedNamespace;
+use DoclerLabs\ApiClientGenerator\Output\Copy\Request\AuthenticationCredentials;
+use PhpParser\Node\Expr;
+
+class BasicAuthenticationSecurityStrategy extends SecurityStrategyAbstract
+{
+    private const         PROPERTY_CREDENTIALS = 'credentials';
+    private const         SCHEME               = 'basic';
+    private const         TYPE                 = 'http';
+
+    private CodeBuilder $builder;
+
+    public function __construct(CodeBuilder $builder)
+    {
+        $this->builder = $builder;
+    }
+
+    public function getImports(string $baseNamespace): ImportCollection
+    {
+        return parent::getImports($baseNamespace)
+            ->add(CopiedNamespace::getImport($baseNamespace, AuthenticationCredentials::class));
+    }
+
+    public function getProperties(Operation $operation, Specification $specification): array
+    {
+        $statements = [];
+
+        if ($this->isAuthenticationAvailable($operation, $specification)) {
+            $statements[] = $this->builder->localProperty(self::PROPERTY_CREDENTIALS, 'AuthenticationCredentials', 'AuthenticationCredentials');
```

Comment:
```suggestion
            $statements[] = $this->builder->localProperty(
                self::PROPERTY_CREDENTIALS,
                'AuthenticationCredentials',
                'AuthenticationCredentials'
            );
```
more readable perhaps?